.gitignore

```diff
@@ -0,0 +1,5 @@
+*.sqlite3
```

Comment:
Sorry i missed. Remove this file. If you remove the database from the repo how will people work.